manifests/charts/base/templates/validatingwebhookconfiguration.yaml

```diff
@@ -9,10 +9,14 @@ metadata:
 webhooks:
   - name: validation.istio.io
     clientConfig:
+      {{- if .Values.base.validationURL }}
```

Comment:
You can add it to Values directly ( until it's part of mesh config ). 

base is specific to operator - we want to support helm3, etc.